recipe/meta.yaml

```diff
@@ -11,17 +11,16 @@ source:
         - EHapi.c.patch
 
 build:
-    number: 1
+    number: 2
     skip: True  # [win]
 
 requirements:
     build:
-        - hdf5
+        - hdf5 1.8.15.1
```

Comment:
:+1: 

Though we desperately need https://github.com/conda/conda-build/pull/747 for this reason.
It is getting frustrating the lack of progress there. :disappointed: